src/memory_targets/memory_target_numa.c

```diff
@@ -189,27 +189,31 @@ static umf_result_t query_attribute_value(void *srcMemoryTarget,
                                           memattr_type_t type) {
     hwloc_topology_t topology = umfGetTopology();
     if (!topology) {
+        LOG_ERR("umfGetTopology() failed");
         return UMF_RESULT_ERROR_NOT_SUPPORTED;
     }
 
     hwloc_obj_t srcNumaNode = hwloc_get_obj_by_type(
         topology, HWLOC_OBJ_NUMANODE,
         ((struct numa_memory_target_t *)srcMemoryTarget)->physical_id);
     if (!srcNumaNode) {
+        LOG_ERR("hwloc_get_obj_by_type() failed");
         return UMF_RESULT_ERROR_INVALID_ARGUMENT;
     }
 
     hwloc_obj_t dstNumaNode = hwloc_get_obj_by_type(
         topology, HWLOC_OBJ_NUMANODE,
         ((struct numa_memory_target_t *)dstMemoryTarget)->physical_id);
     if (!dstNumaNode) {
+        LOG_ERR("hwloc_get_obj_by_type() failed");
         return UMF_RESULT_ERROR_INVALID_ARGUMENT;
     }
 
     // Given NUMA nodes aren't local, HWLOC returns an error in such case.
     if (!hwloc_bitmap_intersects(srcNumaNode->cpuset, dstNumaNode->cpuset)) {
         // Since we want to skip such query, we return the worst possible
         // value for given memory attribute.
+        LOG_ERR("hwloc_bitmap_intersects() failed - memattr_get_worst_value()");
```

Comment:
If `if (!hwloc_bitmap_intersects(srcNumaNode->cpuset, dstNumaNode->cpuset)` condition wasn't met then HWLOC would fail the `hwloc_memattr_get_value` call, but we are aware of that and skip such query. Perhaps `LOG_DEBUG` would be more appropriate.

Reply:
Done